xbmc/cores/AudioEngine/Sinks/AESinkWASAPI.cpp

```diff
@@ -808,86 +807,7 @@ void CAESinkWASAPI::EnumerateDevicesEx(AEDeviceInfoList &deviceInfoList, bool fo
           CLog::Log(LOGNOTICE, __FUNCTION__": sample rate 192khz on device \"%s\" seems to be not supported.", strFriendlyName.c_str());
         }
       }
-
-      /* Test format for channels iteration */
```

Comment:
It seems to be only used for pushing back AE_FMT_LPCM which the Engine does not use at all